app/services/index.js

```diff
@@ -1,2 +1,3 @@
-export { default as voteService } from './topics';
+import voteService from './topics';
```

Comment:
I think this bit caused #784 . Reading [ponyfoo's es6 article](https://ponyfoo.com/articles/es6), [this](https://ponyfoo.com/articles/es6-modules-in-depth#exporting-a-default-binding) and [this](https://medium.com/@kentcdodds/misunderstanding-es6-modules-upgrading-babel-tears-and-a-solution-ad2d5ab93ce0#.chotb1xzh) article. It seems I might have misunderstood ES6 imports and exports

@ZeroCho For now, I think we should either:
1. Revert to what it was before (for this file)
2. `export {}` alongside with `export default {}`. This makes consuming this module confusing as there are several ways to use this module.
3. `import defaultModule` instead of `import { foo } from './bar'`.

```js
import service from '../services';
```

I'm still thinking which is the best approach